src/server/ApiRoutes/Bib.js

```diff
@@ -52,12 +50,28 @@ function bibSearchServer(req, res, next) {
         return res.redirect(`${appConfig.baseUrl}/404`);
       }
 
-      res.locals.data.Store = {
-        bib: data,
-        searchKeywords: req.query.searchKeywords || '',
-        error: {},
-      };
-      next();
+      if (data.subjectLiteral.length) {
```

Comment:
Is it possible for `data.subjectLiteral` not to be defined? Could add a comment here about what we are checking for.